daiquiri/datalink/adapter.py

```diff
@@ -0,0 +1,165 @@
+from django.conf import settings
+from django.urls import reverse
+
+from daiquiri.core.constants import ACCESS_LEVEL_PUBLIC
+from daiquiri.core.adapter import DatabaseAdapter
+from daiquiri.core.utils import import_class, get_doi_url
+
+
+def DatalinkAdapter():
+    return import_class(settings.DATALINK_ADAPTER)()
+
+
+class BaseDatalinkAdapter(object):
+    """
+    Each datalink adapter needs to configure a set of resource types.
+
```

Comment:
I would also add some examples for the `resource_types` , they are not the same for the oaiAdapter, DatalinkAdapater and MetadataAdapter.

```suggestion
resource_types are declare as a list(string), i.e.: [table, schema,...]
```